builders/build/eth-api/libraries/ethersjs.md

```diff
@@ -7,48 +7,54 @@ description: 通过本教程以学习如何使用以太坊EtherJS代码库在Moo
 
 ## 概览 {: #introduction }
 
-[Ethers.js](https://docs.ethers.io/){target=_blank}代码库提供用户使用JavaScript与以太坊节点交互的多样工具，类似于Web3.js。Moonbeam拥有与以太坊相似的API供用户使用，其与以太坊风格的JSON RPC调用完全兼容。因此，开发者可以利用此兼容特性并使用Ethers.js库与Moonbeam节点交互，与在以太坊操作相同。您可以在此[博客文章](https://medium.com/l4-media/announcing-ethers-js-a-web3-alternative-6f134fdd06f3){target=_blank}了解更多关于Ethers.js的信息。
+[Ethers.js](https://docs.ethers.org/){target=_blank}代码库提供用户使用JavaScript与以太坊节点交互的多样工具，类似于Web3.js。Moonbeam拥有与以太坊相似的API供用户使用，其与以太坊风格的JSON RPC调用完全兼容。因此，开发者可以利用此兼容特性并使用Ethers.js库与Moonbeam节点交互，与在以太坊操作相同。您可以在其[官方文档](https://docs.ethers.org/v6/){target=_blank}获取更多关于如何使用Ethers.js的信息。
 
 在本教程中，您将学习如何使用Ethers.js库在Moonbase Alpha上发送交易和部署合约。本教程也同样适用于[Moonbeam](/builders/get-started/networks/moonbeam/){target=_blank}，[Moonriver](/builders/get-started/networks/moonriver/){target=_blank}，或是[Moonbeam development node](/builders/get-started/networks/moonbeam-dev/){target=_blank}。
 
 ## 查看先决条件 {: #checking-prerequisites }
 
 在开始本教程示例之前，您将需要提前准备以下内容：
 
- - 具有拥有一定数量资金的账户。 
+ - 具有拥有一定数量资金的账户。
  --8<-- 'text/faucet/faucet-list-item.md'
- - 
+ -
 --8<-- 'text/common/endpoint-examples.md'
 
 !!! 注意事项
     --8<-- 'text/common/assumes-mac-or-ubuntu-env.md'
 
-## 创建一个JavaScript项目 {: #create-a-javascript-project }
+安装Scaffold-ETH
```

Comment:
Just double checking that this scaffold eth mention is correct? I don't see it on the English page.

Reply:
Haha nope, I added it there to copy the first two characters. Good catch!